articles/active-directory/connect/active-directory-aadconnectsync-operations.md

```diff
@@ -240,7 +240,7 @@ do 
 	{
 		Write-Host Hit the maximum users processed without completion... -ForegroundColor Yellow
 
-		#export the collection of users as as CSV
+		#export the collection of users a CSV
```

Comment:
Could you change this to "users as a CSV" please?  Then I can approve the merge.  Thanks.

Reply:
thanks, must have messed up the switch. Looks like that was what I was going for too 😊